src/fermilib/transforms/_bksf.py

```diff
@@ -0,0 +1,384 @@
+            for r in range(n_qubits):
+                for s in range(n_qubits):
+                    coefficient = complex(iop[p, q, r, s])
+
+                    # Skip zero terms.
+                    if (not coefficient) or (p == q) or (r == s):
+                        continue
+
+                    # Identify and skip one of the complex conjugates.
+                    if [p, q, r, s] != [s, r, q, p]:
+                        if len(set([p, q, r, s])) == 4:
+                            if min(r, s) < min(p, q):
+                                continue
+                        elif p != r and q < p:
+                                continue
+
+                    # Handle the two-body terms.
+                    transformed_term = bksf_two_body(edge_matrix,p, q, r, s)
+                    transformed_term *= coefficient
+                    qubit_operator += transformed_term
+    return qubit_operator
+
+def bksf_edge_matrix(iop,n_qubits=None):
+    """Edge matrix contains the information about the edges between 
+        vertices. Edge matrix is required to build the operators in bksf
+        model.
+        Input: It takes as input interaction operator.
+    Returns:
+        A square numpy array containing information about the edges present in
+        the model
+    """
+    from fermilib.utils import count_qubits
+    if n_qubits is None:
+        n_qubits = count_qubits(iop)
+    if n_qubits < count_qubits(iop):
+        n_qubits = count_qubits(iop)
+    edge_matrix=1j*np.zeros((n_qubits,n_qubits))
+    # Loop through all indices.
+    for p in range(n_qubits):
+        for q in range(n_qubits):
+
+            
+            # Handle one-body terms.
+            coefficient = complex(iop[p, q])
+            if coefficient and p >= q:
+                edge_matrix[p,q]=bool(complex(iop[p,q]))
+                
+
+            # Keep looping for the two-body terms.
+            for r in range(n_qubits):
+                for s in range(n_qubits):
+                    coefficient2 = complex(iop[p, q, r, s])
+
+                    # Skip zero terms.
+                    if (not coefficient2) or (p == q) or (r == s):
+                        continue
+
+                    # Identify and skip one of the complex conjugates.
+                    if [p, q, r, s] != [s, r, q, p]:
+                        if len(set([p, q, r, s])) == 4:
+                            if min(r, s) < min(p, q):
+                                continue
+                        elif p != r and q < p:
+                                continue
+                            
+                    if (p == q) or (r == s):
+                        continue
+            
+                    # Handle case of four unique indices.
+                    elif len(set([p, q, r, s])) == 4:
+                        
+                        if coefficient2 and p >= q:
+                            edge_matrix[p,q]=bool(complex(iop[p,q,r,s]))
```

Comment:
Another pep8 thing. Just use the linter. There are some other violations but I will stop pointing out and let the pep8 linter do the job for me!